tardis/tests/test_rpacket.py

```diff
@@ -0,0 +1,146 @@
+import os
+from sys import float_info as floats
+import random
+from ctypes import CDLL, c_double
+
+import pytest
+import numpy as np
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+MAX_FLOAT, MIN_FLOAT = floats[0], floats[3]
+MAX_INT = 10000
```

Comment:
use `pytest` fixtures for this, then we can parameterize in the future. global variables are a bad idea :wink: